Extension/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Version 1.6.0-insiders2: August 17, 2021
+### Bug Fixes
```

Comment:
What about the debugger fix for https://github.com/microsoft/vscode-cpptools/issues/7971 ?